packages/components/icon/src/icon.scss

```diff
@@ -1,27 +1,53 @@
-$sizes: 2xs, xs, sm, md, lg, xl, 2xl, 3xl, 4xl;
-
 :host {
-  --sl-icon-container-size: var(--sl-size-icon-md);
   --sl-icon-fill-accent: var(--sl-color-palette-accent-300);
   --sl-icon-fill-default: currentcolor;
-  --sl-icon-size: var(--sl-size-icon-md);
+  --sl-icon-size: var(--sl-size-200);
 
-  block-size: var(--sl-icon-container-size, unset);
+  block-size: max(1cap, var(--sl-icon-size));
```

Comment:
It looks like the icons in the number-field are broken
<img width="1132" alt="image" src="https://github.com/user-attachments/assets/4f6b71bb-d213-4b19-83b5-e1989a005980" />

Reply:
Also in the date-field
<img width="650" alt="image" src="https://github.com/user-attachments/assets/a075dcd4-b8a6-4968-af9d-ad29dc6b83a8" />

Reply:
And search-field
<img width="960" alt="image" src="https://github.com/user-attachments/assets/cf6b5903-fc9a-4092-8558-2e915f91dbb8" />